### PR TITLE
Add pipes support

### PR DESCRIPTION
**Pull request by pevloshevskaya**

### CODE REVIEW

File: pkg/snowflake/pipe.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	q.WriteString(`CREATE`)
+
+	q.WriteString(fmt.Sprintf(` PIPE %v`, pb.QualifiedName()))
+
+	if pb.autoIngest {
+		q.WriteString(` AUTO_INGEST = TRUE`)
+	}
+
+	if pb.comment != "" {
+		q.WriteString(fmt.Sprintf(` COMMENT = '%v'`, pb.comment))
```
Comment (ryanking) [inline pkg/snowflake/pipe.go]:
```suggestion
		q.WriteString(fmt.Sprintf(` COMMENT = '%v'`, EscapeString(pb.comment)))
```

We have been lax about escaping values but need to get better.

File: pkg/snowflake/pipe.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	}
+	if pb.copyStatement != "" {
+		q.WriteString(fmt.Sprintf(` AS %v`, pb.copyStatement))
```
Comment (ryanking) [inline pkg/snowflake/pipe.go]: Not sure if this needs to be escaped as well.

File: pkg/snowflake/pipe.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+	}
+
+	return q.String()
+}
+
+// ChangeComment returns the SQL query that will update the comment on the pipe.
+func (pb *PipeBuilder) ChangeComment(c string) string {
+	return fmt.Sprintf(`ALTER PIPE %v SET COMMENT = '%v'`, pb.QualifiedName(), c)
```
Comment (ryanking) [inline pkg/snowflake/pipe.go]:
```suggestion
	return fmt.Sprintf(`ALTER PIPE %v SET COMMENT = '%v'`, pb.QualifiedName(), EscapeString(c))
```

Comment (pevloshevskaya) [reply pkg/snowflake/pipe.go]: I fixed all the comments in [this commit ](https://github.com/chanzuckerberg/terraform-provider-snowflake/pull/128/commits/b34292c2a74c3693501588a9fbd450eb0eb42812)

**Comment by doctoryes** [comment]
🎉 !

**Pull request merged**